fraudserv/service.go

```diff
@@ -28,9 +28,15 @@ var (
 	tracer = otel.Tracer("fraudserv")
 )
 
-// fraudRequests is the amount of external requests that will be tried to get fraud proofs from
-// other peers.
-const fraudRequests = 5
+const (
+	// fraudRequests is the amount of external requests that will be tried to get fraud proofs from
+	// other peers.
+	fraudRequests = 5
+
+	// headThreshold specifies the maximum allowable height of the Proof
+	// relative to the network head to be verified.
+	headThreshold uint64 = 20
```

Comment:
No, this means that in case the network head(not the store head) is 100 - you won't accept proofs with 
height > 120, as this header is not yet exist.